### DISCUSSION TITLE
Only one GPU in a multi-GPU system is used when running Lammps

### DISCUSSION THREAD

**Discussion by yunsuyunsu** [deepmodeling/deepmd-kit#718] [Q&A] [answered]
I installed GPU version DeePMD and Lammps with conda on a system with 8 GPUs. After training, I run "lmp -sf gpu -pk gpu 8 -i in.lammps", but an error shows up "ERROR: Using suffix gpu without GPU package installed (src/lammps.cpp:801)". If I run "lmp -i in.lammps", only 1 out of the 8 GPUs is used. My question is how to use all the 8 GPUs when running lammps.

**Comment by njzjz** [comment, marked as answer]
See #667. The correct way is to use MPI, i.e. `mpirun -n 8 lmp ...`.
